test/unit/search_api_test.rb

```diff
@@ -26,7 +26,7 @@ class SearchAPITest < ActiveSupport::TestCase
 
   context "given a search scoped to a manual" do
     setup do
-      @manual_link = '/manual/manual-name'
+      @manual_link = 'manual/manual-name'
```

Comment:
This line shouldn't be reverted - the value used here is the section's `manual` field which does now have the leading slash, and the test setup data should reflect that. The tests don't make any assertions about this value so they still pass with the slash left in.